sktime/utils/tests/test_mlflow_sktime_model_export.py

```diff
@@ -50,7 +50,7 @@ def mock_s3_bucket():
 
 
 @pytest.mark.skipif(
-    not _check_soft_dependencies("mlflow", severity="none"),
+    not _check_soft_dependencies(severity="none"),
```

Comment:
Why are we not using `_check_mlflow_dependencies` here?

(Same doubt for the below changes in this module.)

---

Also, what happens if `packages` argument in `_check_soft_dependencies` is a tuple of length 0?

Reply:
the logic is "and", so it will just default to `True`, because `True and x` is the same as `x`.

Reply:
in any case, this is a bug you found...

Reply:
Probably a stupid question, but why is it a bug? I asked about 0 length just out of curiosity, is returning `True` not what user will expect?

Reply:
the behaviour of `_check_soft_dependencies` is not a bug as it is mathematically consistent with the definition of conjunction (zero length conjunction is "True" per usual convention).

What I mean, the bug is line 53 that you highlighted, as the skip condition is always `False`.

Reply:
Find and replace brought in this bug, will fix it up.